test/powershell/Modules/Microsoft.PowerShell.Core/Enter-PSHostProcess.Tests.ps1

```diff
@@ -25,11 +25,13 @@ Describe "Enter-PSHostProcess tests" -Tag Feature {
         }
     }
 
-    It "Can enter and exit another PSHost" {
+    # Skip on Appveyor due to PSReadline issue.
+    It "Can enter and exit another PSHost" -Skip:($env:AppVeyor) {
```

Comment:
What is the contents of `$env:AppVeyor`?  The failure indicates that it's a string.  Perhaps this should be:

`-Skip:($env:AppVeyor -ne $null)`